lib/generators/shopify_app/install/templates/shopify_app.rb.tt

```diff
@@ -7,6 +7,9 @@ ShopifyApp.configure do |config|
   config.after_authenticate_job = false
   config.api_version = "<%= @api_version %>"
   config.shop_session_repository = 'Shop'
+
+  config.reauth_on_access_scope_changes = true
```

Comment:
👋 @rezaansyed do you recall why we made this configurable, i.e. what's the use for _not_ enabling it? We're troubleshooting an issue with `https://github.com/Shopify/shopify-graphiql-app` which seems related to this being missing from the config.

(cc @teddyhwang @nelsonwittwer)

Reply:
Wouldn't the app break if the developers didn't automatically reauth? How did this work before this change, developers would have to do an OAuth in their app code themselves?

Reply:
Yup, that's correct

Reply:
Maybe what you want is to remove this from the template and default the value to on, but still support it as a config value so it can be explicitly disabled if an for some reason doesn't want this behaviour?

Reply:
One bespoke case I noticed is that a logged in user may not have the appropriate permission to authorize the scope. 

Not sure if this is the best strategy or if it's already handled in the current implementation, but see here:
https://shashankkumar.medium.com/managing-access-scopes-for-shopify-apps-c93b3cf3541b